mkdocs.yml

```diff
@@ -547,6 +547,12 @@ plugins:
               "faq/troubleshooting/why-cant-i-see-or-add-my-organizations-repositories.md": "organizations/managing-repositories.md"
               "faq/general/which-browsers-does-codacy-support.md": "faq/general/which-platforms-and-technologies-does-codacy-support.md"
               "faq/general/which-version-control-systems-does-codacy-support.md": "faq/general/which-platforms-and-technologies-does-codacy-support.md"
+              "related-tools/codacy-plugin-tools.md": "getting-started/supported-languages-and-tools.md#codacy-plugin-tools"
```

Comment:
I didn't know that the mkdocs-redirects plugin already supported redirecting to URL fragments. That's cool! :nerd_face: